test/e2e/pkg/suite_test.go

```diff
@@ -19,40 +20,32 @@ import (
 )
 
 var (
+	consumerOpts     *ConsumerOptions
 	apiServerAddress string
-	kubeconfig       string
-	consumer_name    string
-	kubeClient       *kubernetes.Clientset
 	apiClient        *openapi.APIClient
 	helper           *test.Helper
```

Comment:
@clyang82 
I'm curious why we need `helper` in e2e testing? It's mainly used by integration test to set up maestro servers, but for e2e testing, the only use of `helper` is to create openapi resource?

Reply:
Yes. only use it to create openapi resource.